code/modules/mob/living/living_defines.dm

```diff
@@ -15,7 +15,7 @@
 	var/toxloss = 0		//Toxic damage caused by being poisoned or radiated
 	var/fireloss = 0	//Burn damage caused by being way too hot, too cold or burnt.
 	var/cloneloss = 0	//Damage caused by being cloned or ejected from the cloner early. slimes also deal cloneloss damage to victims
-	var/brainloss = 0	//'Retardation' damage caused by someone hitting you in the head with a bible or being infected with brainrot.
+	//brainloss moved to be handled on the brain organ itself. Rip.

```

Comment:
This might be a dumb question. If you've moved this: does brainloss show up in the top left of VV still? Or did you remove that too? I've looked over this, and I'm probably missing it, but